> /home/neuro/Downloads/models/cacam_models/camkii/Ng-cam/cam2/5hz_6sub_Ngcam0_cam2_s/rxn_process.py(26)<module>()
-> f.write(rxn_part)
(Pdb) Traceback (most recent call last):
  File "rxn_process.py", line 26, in <module>
    f.write(rxn_part)
  File "rxn_process.py", line 26, in <module>
    f.write(rxn_part)
  File "/home/neuro/miniconda2/lib/python2.7/bdb.py", line 49, in trace_dispatch
    return self.dispatch_line(frame)
  File "/home/neuro/miniconda2/lib/python2.7/bdb.py", line 67, in dispatch_line
    self.user_line(frame)
  File "/home/neuro/miniconda2/lib/python2.7/pdb.py", line 158, in user_line
    self.interaction(frame, None)
  File "/home/neuro/miniconda2/lib/python2.7/pdb.py", line 210, in interaction
    self.cmdloop()
  File "/home/neuro/miniconda2/lib/python2.7/cmd.py", line 130, in cmdloop
    line = raw_input(self.prompt)
IOError: [Errno 9] Bad file descriptor
110
115
120
125
130
135
140
145
150
155
160
165
170
175
180
185
190
195
321
326
340
352
361
368
374
384
389
396
402
408
413
420
425
431
439
444
449
454
459
467
478
483
488
497
502
510
522
529
534
540
545
550
557
564
569
574
579
584
589
596
601
606
611
616
622
628
635
640
645
650
655
660
665
673
680
685
692
700
705
710
715
721
726
733
740
745
750
755
760
766
771
778
783
788
793
799
804
809
815
820
826
831
836
842
847
852
857
862
867
872
878
883
888
893
898
905
911
916
921
929
934
939
945
950
955
961
967
972
977
982
987
993
998
1003
1010
1015
1020
1025
1033
1040
1045
1050
1055
1061
1066
1071
1080
1085
1090
1097
1102
1107
1143
1156
1161
1166
1173
1178
1184
1189
1203
1211
1216
1221
1226
1231
1236
1242
1247
1262
1283
1288
1293
1298
1305
1313
1318
1324
1329
1334
1339
1361
1366
1371
1381
1386
1391
1415
1428
1433
1438
1443
1448
1453
1458
1463
1487
1492
1532
